contracts/interchain-token-service/src/contract.rs

```diff
@@ -36,14 +36,18 @@ pub enum Error {
     UnfreezeChain,
     #[error("failed to set chain config")]
     SetChainConfig,
-    #[error("failed to query its address")]
-    QueryItsContract,
-    #[error("failed to query all its addresses")]
-    QueryAllItsContracts,
     #[error("failed to disable execution")]
     DisableExecution,
     #[error("failed to enable execution")]
     EnableExecution,
+    #[error("failed to query its address")]
+    QueryItsContract,
+    #[error("failed to query all its addresses")]
+    QueryAllItsContracts,
+    #[error("failed to query a specific token instantiation")]
```

Comment:
Why `instantiation`?

Reply:
My brain just went with the wrong word. I meant instance